cmd/influxd/run.go

```diff
@@ -187,6 +188,31 @@ func Run(config *Config, join, version string) (*messaging.Broker, *influxdb.Ser
 			}
 		}
 
+		// Spin up any OpenTSDB servers
+		if config.OpenTSDB.Enabled {
```

Comment:
My mistake @tcolgate -- I must have misinterpreted the indentation. Looks good -- thanks.